controllers/nificluster_controller.go

```diff
@@ -149,7 +149,7 @@ func (r *NifiClusterReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 			case errorfactory.ResourceNotReady:
 				r.Log.Info("A new resource was not found or may not be ready", zap.String("reason", err.Error()))
 				return reconcile.Result{
```

Comment:
This `reconcile.Result` can actually be replaced with the helper function `RequeueAfter(intervalNotReady)`

Reply:
I've added that conversion over to this PR instead 😄